### PR TITLE
SPARQL DPUs: Documentation update

### LINKED ISSUE #332: e-sparqlEndpoint: Typo in Slovak localization
In the Slovak locale for e-sparqlEndpoint the endpoint URL is written as "Enpoint URL". This should be corrected, or even better, made consistent with the label in l-filesToParliament, which is "URL koncového bodu".

### PR DESCRIPTION
**Pull request by mbl-eea-sk**
Slovak documentations of all t-Sparql DPUs have been purified from English expresses and their stylizations unified.
Resolves #332

### CODE REVIEW

File: t-sparqlConstruct/src/main/resources/resources_sk.properties [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 
-SparqlConstructVaadinDialog.constructQuery = SPARQL dotaz typu CONSTRUCT
+SparqlConstructVaadinDialog.constructQuery = SPARQL kon\u0161truk\u010Dn\u00FD dotaz
 SparqlConstructVaadinDialog.emptyQuery     = Dotaz nesmie by\u0165 pr\u00E1zdny
 SparqlConstructVaadinDialog.perGraphMode   = P\u00FA\u0161\u0165anie dotazu po jednom grafe
 
-dpu.name      = RDF transform\u00E1cia pomocou SPARQL construct
-dpu.name.menu = SPARQL Construct
+dpu.name = RDF transform\u00E1cia pomocou SPARQL construct
+dpu.name.menu = t-sparqlConstruct
```
Comment (tomas-knap) [inline t-sparqlConstruct/src/main/resources/resources_sk.properties]: @mbl-eea-sk Did you discuss this change with @eea04 ? I think that in eDem project they do not want t-sparqlConstruct, but rather SPARQL Construct as the DPU menu name.

Comment (mbl-eea-sk) [inline t-sparqlConstruct/src/main/resources/resources_sk.properties]: There were differences between Slovak and English version so I took the English one as referential. @eea04 should thie be keeped as it was
(English = t-sparqlConstruct, Slovak = SPARQL Construct)
or perhaps the English resources should be updated by the Slovak ones?

**Comment by tomas-knap** [comment]
I changed the names in EN version as it was before - this change would be inconsistent with other plugins which are named (in EN version) according to their name in pom.xml

**Comment by tomas-knap** [comment]
Can be merged when the issue with SK menu name change is solved.

File: t-sparqlConstruct/src/main/resources/resources_sk.properties (hunk 1 above)
Comment (tomas-knap) [inline t-sparqlConstruct/src/main/resources/resources_sk.properties]: English menu names should be kept as they were. The only question is whether it is correct to update SK menu names from "SPARQL Construct" to "t-sparqlConstruct", I think no.

**Pull request merged**